google/cloud/internal/background_threads_impl.cc

```diff
@@ -13,14 +13,28 @@
 // limitations under the License.
 
 #include "google/cloud/internal/background_threads_impl.h"
+#include <algorithm>
 
 namespace google {
 namespace cloud {
 inline namespace GOOGLE_CLOUD_CPP_NS {
 namespace internal {
 
 AutomaticallyCreatedBackgroundThreads::AutomaticallyCreatedBackgroundThreads()
-    : runner_([](CompletionQueue cq) { cq.Run(); }, cq_) {}
+    : AutomaticallyCreatedBackgroundThreads(1, {}) {}
+
+AutomaticallyCreatedBackgroundThreads::AutomaticallyCreatedBackgroundThreads(
+    std::size_t thread_count)
+    : AutomaticallyCreatedBackgroundThreads(
+          thread_count == 0 ? 1 : thread_count, {}) {}
+
+AutomaticallyCreatedBackgroundThreads::AutomaticallyCreatedBackgroundThreads(
+    std::size_t thread_count, Normalized)
+    : pool_(thread_count) {
```

Comment:
Is this where the `thread_count == 0 ? 1 : thread_count` would be better applied?

Reply:
Done.